juniper_official/Solutions/EVPN-VXLAN/spine-evpn-vxlan-kpis.playbook

```diff
@@ -0,0 +1,74 @@
+/*
+ * Playbook contains multiple rules which monitors Data Center Spine
+ * nodes and notifies when anomalies are found.
+ *
+ * 1) Rule evpn-advertised-prefix.rule, detects the advertised route count
+ *    threshold breaches and notify anomalies.
+ * 2) Rule evpn-received-prefix.rule, detects the received route count
+ *    threshold breaches and notify anomalies.
+ * 3) Rule evpn-route-count.rule, detects the route count threshold
+ *    breaches and notify anomalies.
+ * 4) Rule evpn-mac-count.rule, detects the evpn mac count threshold
+ *    breaches and notify anomalies.
+ * 5) Rule bfd-session-state.rule, detects BFD session state change and
+ *    notifies anomalies when error count increases.
+ * 6) Rule vtep-interface-count.rule, detects the vtep interface  count
+ *    threshold breaches and notify anomalies.
+ * 7) Rule vtep-interface-status.rule, detects the vtep interface status
+ *    notify anomalies.
+ * 8) Rule vtep-interface-traffic.rule,monitors interface traffic and notifies
+      anomalies when traffic exceeds threshold.
+ * 9) Rule vtep-interface-flap.rule, detects interface flaps and notifies
+ *    anomalies when link flap count increases.
+ * 10) Rule vtep-interface-error.rule, detects interface errors and notifies
+ *     anomalies when error count increases.
+ * 11) Rule vtep-tunnel-statistics.rule, detects logical interface statisticsr
+ *     and notifies anomalies when error count increases.
+ * 12) Rule irb-interface-status.rule, detects the irb interface status
+ *     notify anomalies.
+ * 13) Rule irb-interface-traffic.rule,monitors interface traffic and notifies
+ *     anomalies when traffic exceeds threshold.
+ * 14) Rule irb-next-hops.rule, monitors irb interface next-hops and notifies
+ *     anomalies when exceeds threshold.
+ * 15) Rule interface-in-errors-iagent.rule, monitors interface input errors 
+ *     and notifies anomalies when exceeds threshold.
+ * 16) Rule interface-out-errors-iagent.rule, monitors interface output errors
+ *     and notifies anomalies when exceeds threshold.
+ * 17) Rule interface-in-traffic-iagent.rule, monitors interface input traffic
+ *     and notifies anomalies when exceeds threshold.
+ * 18) Rule interface-out-traffic-iagent.rule, monitors interface output traffic
+ *     and notifies anomalies when exceeds threshold. 
+ * 19) Rule interface-flaps-iagent.rule, monitors interface flaps and notifies
+ *     anomalies when exceeds threshold. 
+ * 20) Rule interface-status-iagent.rule, monitors interface status and notifies
+ *     anomalies when exceeds threshold. 
+ * 21) Rule bgp-neighbor-state-iagent.rule, monitors bgp state and notifies
+ *     anomalies 
```

Comment:
s/notifies anomalies/notifies anomaly when neighbor state is down/g

Reply:
updated the rule